### DISCUSSION TITLE
How to add navigation item to the top navigation bar in the Admin theme?

### DISCUSSION THREAD

**Discussion by MikeAlhayek** [OrchardCMS/OrchardCore#8191] [Q&A]
How can I hook into the admin theme and add more admin. I am aware that implementing `INavigationProvider` will allow me to add menu items to a navigation menu. But the top section does not seem to have Navigation menu. How can I add menu on the top navigation bar?

![image](https://user-images.githubusercontent.com/24724371/103388684-4ca4a680-4abf-11eb-958c-1ae7c0777da9.png)

**Comment by Skrypt** [comment]
It's a zone which can be filled from any admin view.
Search for `@await RenderSectionAsync("NavbarTop", required: false)`
It's in TheAdmin theme.

**Reply by MikeAlhayek** [reply]
I found it. But how do I create a menu into that zone? I created a a menu using the GUI, but how would I associate that menu to the `NavbarTop` zone?

**Reply by Skrypt** [reply]
You would need to create your own MenuProvider. There's none as of now for this.
We add single fixed links on the top navbar for now and the NavbarTop is a free zone for anyone to use.
You can do a `<zone Name="NavbarTop">Your Content</zone>` "I think". Just like the page title is added to the top navbar.

Maybe a shape assigned to this zone would work too.
I can't be more explicit, it would take too much time. You will need to try and see 😉
